core/src/main/java/org/geysermc/geyser/translator/level/block/entity/SpawnerBlockEntityTranslator.java

```diff
@@ -69,8 +69,8 @@ public void translateTag(NbtMapBuilder builder, CompoundTag tag, int blockState)
 
         CompoundTag spawnData = tag.get("SpawnData");
         if (spawnData != null) {
-            StringTag idTag = ((CompoundTag) spawnData.get("entity")).get("id");
-            if (idTag != null) {
+            CompoundTag entityTag = (spawnData.get("entity"));
```

Comment:
Spare paranthesis here?

Reply:
oops, forgot to remove it. I've changed the logic though to just replace the spawner altogether, since that won't spawn particles for all empty spawners